### PR TITLE
Fluent: add decorators for loading and completion states

### PR DESCRIPTION
**Pull request by OEvgeny**
### Added

- Added decorators for loading and completion states to the Fluent theme, in PR [#5269](https://github.com/microsoft/BotFramework-WebChat/pull/5269), by [@OEvgeny](https://github.com/OEvgeny)

## Description

This PR enhances the Fluent theme by introducing decorators for loading and completion states of activities.

## Design

Added two new components:
1. `BorderLoader`: Displays an animated loading bar for activities in the preparation state.
2. `BorderFlair`: Shows a colorful animated border for activities in the completion state.

Then `WebChatDecorator` applies them based on the activity's livestreaming state. The implementation uses CSS animations and custom properties for flexibility and performance.

## Specific Changes

- Added new files:
  - `BorderFlair.module.css` and `BorderFlair.tsx`
  - `BorderLoader.module.css` and `BorderLoader.tsx`
- Updated `Decorator.tsx` to use new `BorderFlair` and `BorderLoader` components
- Removed old decorator files (`Flair.tsx`, `Loader.tsx`, etc.)
- Updated `FluentThemeProvider.tsx` to wrap children with `WebChatDecorator`
- Added new CSS variables for animation colors in `Theme.module.css`
- Added test file `withBuiltinDecorator.html` for visual testing

## -

-  [x] I have added tests and executed them locally
-  [x] I have updated `CHANGELOG.md`
-  [ ] ~I have updated documentation~

## Review Checklist

> This section is for contributors to review your work.

-  [ ] Accessibility reviewed (tab order, content readability, alt text, color contrast)
-  [ ] Browser and platform compatibilities reviewed
-  [ ] CSS styles reviewed (minimal rules, no `z-index`)
-  [ ] Documents reviewed (docs, samples, live demo)
-  [ ] Internationalization reviewed (strings, unit formatting)
-  [ ] `package.json` and `package-lock.json` reviewed
-  [ ] Security reviewed (no data URIs, check for nonce leak)
-  [ ] Tests reviewed (coverage, legitimacy)

### CODE REVIEW

File: packages/fluent-theme/src/components/decorator/private/BorderLoader.module.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+@keyframes borderAnimation-position {
+  0% {
+    width: 33%;
+    left: -33%;
```
Comment (compulim) [inline packages/fluent-theme/src/components/decorator/private/BorderLoader.module.css:4]:
```suggestion
    left: -33%;
    width: 33%;
```

File: packages/fluent-theme/src/components/decorator/private/BorderLoader.module.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  }
+
+  100% {
+    width: 33%;
+    left: 100%;
```
Comment (compulim) [inline packages/fluent-theme/src/components/decorator/private/BorderLoader.module.css:9]:
```suggestion
    left: 100%;
    width: 33%;
```

File: packages/api/src/decorator/private/createDecoratorComposer.tsx [hunk 3]
```diff
@@ -10,8 +10,8 @@ type DecoratorMiddlewareInit = typeof activityBorderDecoratorTypeName;
 
 export type DecoratorComposerComponent = (
   props: Readonly<{
-    children?: ReactNode;
-    middleware?: DecoratorMiddleware[];
+    children: ReactNode;
```
Comment (compulim) [inline packages/api/src/decorator/private/createDecoratorComposer.tsx:13]: I regretted... maybe let devs decide if they want to hide the original activity and just showing the decoration.

```suggestion
    children?: ReactNode | undefined;
```

File: packages/fluent-theme/src/components/decorator/private/BorderLoader.module.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  border-radius: inherit;
+}
+
+:global(.webchat-fluent) .border-loader__track {
+  --webchat-decorator-borderLoader-borderSize: var(--webchat-borderLoader-borderSize, 4px);
+
+  background-color: var(--webchat-colorNeutralBackground6);
+  height: var(--webchat-decorator-borderLoader-borderSize);
+  width: 100%;
+  position: relative;
```
Comment (compulim) [inline packages/fluent-theme/src/components/decorator/private/BorderLoader.module.css:23]:
```suggestion
  position: relative;
  width: 100%;
```

File: packages/fluent-theme/src/private/FluentThemeProvider.tsx [hunk 5]
```diff
@@ -38,7 +39,7 @@ const FluentThemeProvider = ({ children, variant = 'fluent' }: Props) => (
     <WebChatTheme>
       <TelephoneKeypadProvider>
         <ThemeProvider activityMiddleware={activityMiddleware} sendBoxMiddleware={sendBoxMiddleware}>
-          {children}
+          <WebChatDecorator>{children}</WebChatDecorator>
```
Comment (compulim) [inline packages/fluent-theme/src/private/FluentThemeProvider.tsx:42]: Maybe call it `<FluentThemeDecorator>`?

**Review by compulim — DISMISSED** [review]

File: packages/fluent-theme/src/private/FluentThemeProvider.tsx (hunk 5 above)
Comment (OEvgeny) [reply packages/fluent-theme/src/private/FluentThemeProvider.tsx:42]: Good call, the name left from the whitelabel unfinished work

**Review by compulim — APPROVED** [review]

**Pull request merged**